services/collectibles/manager.go

```diff
@@ -0,0 +1,99 @@
+package collectibles
```

Comment:
Why do you need Manager layer ? What is the idea ?

Reply:
In order not to increase the Service interface.
Also, the Manager can be passed to the service directly (if it will be necessary) and it can be mocked